### DISCUSSION TITLE
Deezer progress bar

### DISCUSSION THREAD

**Discussion by RiiKi-Kenway** [ShankarBUS/ModernFlyouts#865] [Ideas]
I have encountered some troubles with Deezer app. The songs don't synchronize with the progress bar.

**Comment by prakharb5** [comment]
An explanation for the issue would help. What's happening, what should happen, etc.

**Reply by RiiKi-Kenway** [reply]
Always when i start some music on deezer, the progress bar of the songs remains in 0:00 to 0:00
![screenshot](https://user-images.githubusercontent.com/51724489/140433203-197c343a-1b8b-4e5f-af4d-7b76ed703179.png)

**Reply by prakharb5** [reply]
Ah, ok so, this music app does not support the timeline feature. You can try asking the devs to add that feature, but it doesn't seem likely.
